internal/app/handler/committer/committer.go

```diff
@@ -150,7 +150,8 @@ func (c *Committer) commitBlob(blobsDir string) (map[string]string, error) {
 
 	logger.Info(c.context, "committed blob data")
 	logger.DebugWithFields(c.context, "blob file names", map[string]interface{}{
-		"files": files,
+		"files":    files,
+		"blobURIs": blobURIs,
```

Comment:
is this a potential security leak i.e. storing authenticated URLs in logs?

Reply:
Yes, removed. Left over from debugging :(

Reply:
Good spot